cmds/mount/mount.go

```diff
@@ -35,10 +36,19 @@ func main() {
 	}
 	dev := a[0]
 	path := a[1]
+	var data []string
+	for _, o := range strings.Split(*opt, ",") {
+		f, ok := opts[o]
+		if !ok {
+			data = append(data, o)
+			continue
+		}
+		flags |= f
+	}
 	if *ro {
 		flags |= unix.MS_RDONLY
 	}
-	if err := unix.Mount(a[0], a[1], *fsType, flags, *data); err != nil {
+	if err := unix.Mount(a[0], a[1], *fsType, flags, strings.Join(data, ",")); err != nil {
```

Comment:
Is this what mount does upstream? I think the comma is just convention, not required by the syscall.